src/user/dto/create-user.dto.ts

```
@@ -0,0 +1,14 @@
+import { ApiProperty } from "@nestjs/swagger";
```

ESLint

[error] Replace `"@nestjs/swagger"` with `'@nestjs/swagger'`

(prettier/prettier)